api/app/tests/apps/api/v1/test_filters.py

```diff
@@ -995,13 +995,29 @@ def _request_filter_signals(self, filter_params: dict):
 
         return ids
 
-    def test_retrieve_all_parents_with_changes_in_children(self):
+    def test_retrieve_all_parents_with_changes_in_one_of_the_children(self):
         now = timezone.now()
-        with freeze_time(now - timedelta(hours=1)):
-            parent_signal = SignalFactory()
 
-        with freeze_time(now):
-            SignalFactory(parent=parent_signal)
+        parent_signal = SignalFactory.create()
+        for hour in range(5):
+            # Create 4 child signals 1 hour apart
+            with freeze_time(now + timedelta(hours=hour)):
+                SignalFactory.create(parent=parent_signal)
+
+        with freeze_time(now + timedelta(hours=3)):
+            # This way we have 1 child signal changed after the last update on the parent signal
+            parent_signal.save()
```

Comment:
These children were created after the parent signal, the `auto_now` on `updated_at` will make their `updated_at` greater than the `updated_at` of the parent signal. Either update the parent (timestamped after the children were updated) or remove the preceding 3 lines lines as redundant.

Reply:
That is exactly what this is doing. So no need to change this at all. We created a parent signal, added 5 children over all 1 hour apart from each other and after that updated the parent with the time freezed 1 hour before the last child has been "created/updated".

Reply:
Point is: remove these lines test still works. This is because all of the children have more recent `updated_at` timestamps than the parent.

To get the behavior described in the comment: Save the parent such that it has an `updated_at` timestamp later than the creation of all children --- then update 1 child to be later still.

Reply:
Changed the `signal_to_keep.save()` to `Signal.actions.update_status(data={'text': 'test', 'state': 'b'}, signal=parent_signal_to_keep)`